src/main/java/edu/kis/powp/jobs2d/features/DrawerFeature.java

```diff
@@ -1,12 +1,14 @@
 package edu.kis.powp.jobs2d.features;
 
 import edu.kis.powp.appbase.Application;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.adapter.PanelDriverAdapter;
 import edu.kis.powp.jobs2d.events.SelectClearPanelOptionListener;
 import edu.kis.legacy.drawer.panel.DrawPanelController;
 
 public class DrawerFeature {
 
-	private static DrawPanelController drawerController;
+	private static final PanelDriverAdapter panelDrawerAdapter = new PanelDriverAdapter();
```

Comment:
Drawing in a separate panel issue not fixed correctly. Introduced unnecessary coupling with PanelDriverAdapter (and with its particular instance)